src/style.css

```diff
@@ -0,0 +1,103 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf),
+    url(/fonts/Avenir-Heavy.ttf),
+    url(/fonts/Avenir.ttc);
+  font-weight: 900;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  padding: 20px 8px 8px;
+}
+
+.form__search--big {
+  padding: 23px 0;
+}
+
+.form__search--big:hover {
+  box-shadow: 0 4px 4px 0 rgba(0,0,0, 0.25);
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)